### DISCUSSION TITLE
Why I cannot install mmengin?

### DISCUSSION THREAD

**Discussion by qiansheng2** [open-mmlab/mmengine#1106] [Q&A]
安装mmengin时失败了。
请问有什么建议吗？

![image](https://user-images.githubusercontent.com/5198358/234179857-c9c6f585-b530-46ec-84d2-e19e2205678c.png)

**Comment by qiansheng2** [comment]
我的机器是macbook m1jax,软件是ventura13.3.1

**Comment by zhouzaida** [comment]
请问你这边的安装命令是什么

**Reply by qiansheng2** [reply]
1  安装先用官网上的： python3.7 ，安装失败了
<img width="882" alt="image" src="https://user-images.githubusercontent.com/5198358/234758455-73c9056d-9bf1-4b4f-8e80-c9cf77eea4ff.png">

2 就自行安装了python3.10:
<img width="426" alt="image" src="https://user-images.githubusercontent.com/5198358/234758588-dc03f93a-aefc-46f0-8183-3a055c659b6d.png">
